utils/litemap/examples/language_names_lite_map.rs

```diff
@@ -0,0 +1,45 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+// LiteMap is intended as a small and low-memory drop-in replacement for
+// HashMap. This example demonstrates its use.
+
+#![no_main] // https://github.com/unicode-org/icu4x/issues/395
+
+icu_benchmark_macros::static_setup!();
+
+use icu_locid::subtags::Language;
+use icu_locid_macros::language;
+use litemap::LiteMap;
+
+const DATA: [(Language, &'static str); 11] = [
+    (language!("ar"), "Arabic"),
+    (language!("bn"), "Bangla"),
+    (language!("ccp"), "Chakma"),
+    (language!("en"), "English"),
+    (language!("es"), "Spanish"),
+    (language!("fr"), "French"),
+    (language!("ja"), "Japanese"),
+    (language!("ru"), "Russian"),
+    (language!("sr"), "Serbian"),
+    (language!("th"), "Thai"),
+    (language!("tr"), "Turkish"),
+];
+
+#[no_mangle]
+fn main(_argc: isize, _argv: *const *const u8) -> isize {
```

Comment:
if this is going to be in examples, we should add a comment mentioning that we're doing this for wasm

Reply:
There is a comment explaining the lack of the standard `main` function on line 8 of every example file:

```
#![no_main] // https://github.com/unicode-org/icu4x/issues/395
```

Reply:
ah sorry missed that